web-test-runner.config.ts

```diff
@@ -147,12 +147,13 @@ const testRunnerHtml = (
       globalThis.testRunScript = '${testFramework}';
     </script>
   </head>
-  <body class="sbb-disable-animation">${preloadedIcons
-    .map(
-      (i) => `
+  <body class="sbb-disable-animation">
+    <div>${preloadedIcons
```

Comment:
div seems not to work with testing framework. Also, to prevent possible problems with visual tests, I'd recommend to use "hidden" on it. (I think some scrolling tests sometimes are influences by other parts of the page, not only the actual div where it is tested)

Reply:
No, it works, I just forgot a closing >. I also added the hidden attribute.